### PR TITLE
added window dpi context

### PR DESCRIPTION
**Pull request by henrypp**
Added dpi context information for WindowsExplorer

### CODE REVIEW

**Comment by henrypp** [comment]
Here is error:

>LNK2001 unresolved external symbol __imp_PhGetDllBaseProcedureAddress

**Comment by henrypp** [comment]
sshot-001

**Comment by dmex** [comment]
> unresolved external symbol __imp_PhGetDllBaseProcedureAddress

The plugins use PhGetProcedureAddress instead of PhGetDllBaseProcedureAddress

**Comment by henrypp** [comment]
Done

**Review by dmex — APPROVED** [review]

File: plugins/WindowExplorer/wndprp.c [hunk 1]
```diff
@@ -902,6 +903,96 @@ VOID WepRefreshAutomationProvider(
...
 
+VOID WepRefreshDpiContext(
+    _In_ PWINDOW_PROPERTIES_CONTEXT Context
+    )
+{
+    static PH_INITONCE InitOnce = PH_INITONCE_INIT;
+    static DPI_AWARENESS_CONTEXT (WINAPI* GetWindowDpiAwarenessContext_I)(_In_ HWND hwnd) = NULL;
+    static BOOL (WINAPI* AreDpiAwarenessContextsEqual_I)(_In_ DPI_AWARENESS_CONTEXT dpiContextA, _In_ DPI_AWARENESS_CONTEXT dpiContextB) = NULL;
+
+	DPI_AWARENESS_CONTEXT dpiContext;
```
Comment (jxy-s) [inline plugins/WindowExplorer/wndprp.c:914]: nit: intent

**Review by jxy-s — APPROVED** [review]

**Pull request merged**